### DISCUSSION TITLE
clean images

### DISCUSSION THREAD

**Discussion by kosssi** [outline/outline#5858] [Ideas]
Maybe it's interesting to have a command or API to delete unused images.

PS: Thanks for the new feature to store image on [file storage](https://docs.getoutline.com/s/hosting/doc/file-storage-N4M0T6Ypu7) ;)

**Comment by tommoor** [comment]
This already exists, it is run as part of the cron process which you might not have setup.

https://docs.getoutline.com/s/hosting/doc/scheduled-jobs-RhZzCt770H

**Reply by wohenbushuang** [reply]
seems that the empty folders won't be removed (both the doc's and the attachment's)
